README.rst

```diff
@@ -89,14 +89,19 @@ Main features
 Easy to use
 ~~~~~~~~~~~
 
-Initialize a new backup repository and create a backup archive::
+Initialize a new, password-protected backup repository and create a backup archive::
 
-    $ borg init /path/to/repo
+    $ borg init -e repokey-blake2 /path/to/repo
+    Enter new passphrase:
+    Enter same passphrase again:
+    Do you want your passphrase to be displayed for verification? [yN]: N
```

Comment:
well, maybe we can omit the passphrase "dialogue" for brevity here (and also below).

this is the readme, it should stay relatively short.

Reply:
OK, do whatever you want. I'm done